### ISSUE TITLE
extending to a multiplatform support would be awesome

### ISSUE THREAD

**Issue by ghost** [LeeBamberTGC/Dark-Basic-Pro#9]
would be great seeing Dark-Basic-Pro running on GNU/Linux, OSX, BSD, Haiku, ArOS, Plan9, OpenIndiana, Android, UBports, etc.
how soon can we see it happening?
or will be a project fork needed instead of extending?
and i guess that directx support needs to be deprecated for allowing an easier multiplatform support?

**Comment by Traductus** [comment]
That's correct. Dropping DirectX from DBPro would mean to rewrite the entire solution. That's far from happening at the moment. However, The Game Creators, developers of DBPro, already offer said solution. Its name is AppGameKit or AGK.

Check it out. It might be what you are looking for.
https://www.appgamekit.com/

Here are a few more related links you might find useful:

AppGameKit Chat
https://forum.thegamecreators.com/board/41

Newcomers AppGameKit Corner
https://forum.thegamecreators.com/board/55

AppGameKit Showcase
https://forum.thegamecreators.com/board/48

Have a good one!

**Comment by Cleasonsook** [comment]
@nitrofurano AGK2 it's getting closer to ware DarkBasic Pro was and with more option than DBP had
